src/modules/0.0.12/aws_iam/index.ts

```diff
@@ -133,11 +133,20 @@ export const AwsIamModule: Module2 = new Module2({
       return  arn.split('/').pop();
     },
     rolePolicyComparison: (a: any, b: any) => {
-      return Object.is(Object.keys(a).length, Object.keys(b).length)
-      && Object.is(a.Statement?.length, b.Statement?.length)
-      && a.Statement.every((as: any) => !!b.Statement.find((
-        bs: any) => Object.is(as.Effect, bs.Effect) && Object.is(as.Action, bs.Action) &&
-        Object.is(JSON.stringify(as.Principal), JSON.stringify(bs.Principal))))
+      if (Object.is(a, b)) return true;
+      if (Object.is(a, null) || Object.is(a, null) || !Object.is(typeof a, 'object') || !Object.is(typeof b, 'object')) return false;
```

Comment:
It may also make sense to pull a dependency here. "Deep" equality checking has lots of weird corner cases in JS and [this library](https://lodash.com/docs#isEqual) has already done it.

Reply:
I've installed and tested and it worked, but not sure if is better to add our own code or add new dependencies

Reply:
I think I prefer using lodash’s method here and probably using it in other equality comparators as well to make sure we don’t make mistakes here?